tasks/__init__.py

```diff
@@ -958,3 +958,40 @@ def clean(verbose=False):
 @task(default=True)
 def usage():
     run('invoke --list')
+
+
+### Maintenance Tasks ###
+
+@task
+def set_maintenance_state(start=None, end=None):
```

Comment:
Minor: for consistency in naming (and less typing), `set_maintenance` is fine for the command name. Same with the other task names.